### PR TITLE
Update LSP6 Permission Guide

### PR DESCRIPTION
**Pull request by fhildeb**
Related code snippet:
- [ ] https://github.com/lukso-network/lukso-playground/pull/43

### CODE REVIEW

File: docs/learn/expert-guides/key-manager/get-controller-permissions.md [hunk 1]
```diff
@@ -5,185 +5,169 @@ sidebar_position: 1
...
-We will use the [_erc725.js_](../../../tools/erc725js/getting-started.md) library to do this with just over 30 lines of code.
+:::tip Code repository
+
+You can find the scripts of this guide within our [`lukso-playground`](https://github.com/lukso-network/lukso-playground) repository.
+
+:::
 
 ## Setup
 
+First, we will install the 🛠️[`erc725.js`](../../../tools/erc725js/getting-started.md) library to easily query and decode the 🗂️[ERC725Y data storage](../../../standards/lsp-background/erc725.md#erc725y-generic-data-keyvalue-store) of the Universal Profile. You can then initialize a Universal Profile and connect to the LUKSO L16 network.
```
Comment (Hugoo) [inline docs/learn/expert-guides/key-manager/get-controller-permissions.md:23]: L16 network, really?

**Review by Hugoo — CHANGES_REQUESTED** [review]
i think this needs some more time - i saw a "L16 network" occurence - i guess this was done too fast.

File: docs/learn/expert-guides/key-manager/get-controller-permissions.md [hunk 2]
```diff
@@ -5,185 +5,169 @@ sidebar_position: 1
...
 import LSP6Schema from '@erc725/erc725.js/schemas/LSP6KeyManager.json';
-import Web3 from 'web3';
-
-// setup
-const myUniversalProfileAddress = '0xC26508178c4D7d3Ad43Dcb9F9bb1fab9ceeD58B5';
-const RPC_ENDPOINT = 'https://rpc.testnet.lukso.network';
 
-// step 1 - setup erc725.js
-const web3 = new Web3(RPC_ENDPOINT);
+// Set up Universal Profile on the LUKSO Testnet
```
Comment (Hugoo) [inline docs/learn/expert-guides/key-manager/get-controller-permissions.md:33]: This is not really correct - it doesn't setup anything on the testnet. Maybe more

"Instantiate erc725.js with a Universal Profile address" or smtgh makes more sense

File: docs/learn/expert-guides/key-manager/get-controller-permissions.md [hunk 3]
```diff
@@ -5,185 +5,169 @@ sidebar_position: 1
...
-2. querying each index of the `AddressPermissions[index]` array to retrieve each address individually.
+  if (!result) {
+    console.error('No controllers listed under this Universal Profile ');
+  }
+}
+```
 
-Thanks to _erc725.js_, we can do that very easily. The library will do both steps for us and will returns the full list of controller addresses.
+<Details>
+    <summary>Show <code>AddressPermissions[]</code> value</summary>
```
Comment (Hugoo) [inline docs/learn/expert-guides/key-manager/get-controller-permissions.md:59]: I would keep this close to the code, it is the value of "result" (which btw could be renames to "permissions"

image

The output is quite small so im not sure we even need a Details box.

Btw, please use `<details>` as the issue with this is gone (revert to previous version of react mdx)

Comment (Hugoo) [reply docs/learn/expert-guides/key-manager/get-controller-permissions.md:59]: im not sure these emojis 🗂️ are really needed here, same for 🛠️

File: docs/learn/expert-guides/key-manager/get-controller-permissions.md [hunk 4]
```diff
@@ -5,185 +5,169 @@ sidebar_position: 1
...
+  "SUPER_DELEGATECALL": false,
+  "DELEGATECALL": false,
+  "DEPLOY": true,
+  "SUPER_SETDATA": true,
+  "SETDATA": true,
+  "ENCRYPT": true,
+  "DECRYPT": true,
+  "SIGN": true,
+}
+```
```
Comment (Hugoo) [inline docs/learn/expert-guides/key-manager/get-controller-permissions.md:171]: is this js code? 🤔 

image

image

**Comment by fhildeb** [comment]
Thanks for the feedback. I should not have missed the L16 appearance. Will remind myself to do better self-reviews.

Applied all suggestions, changed naming, and included small outputs into the code 👍🏻
Also updated the related playground code ✅

**Review by Hugoo — APPROVED** [review]

**Pull request merged**